src/Result.ts

```diff
@@ -28,6 +29,9 @@ export async function fromPage(page: Page, story: ProcessedStory): Promise<Resul
   const disabledRules = [...defaultDisabledRules, ...storyDisabledRules];
   const result = await analyze(page, disabledRules);
 
+  const sarifFileName = `${story.storybookId}.sarif`;
+  await exportAxeAsSarifTestResult(sarifFileName, result);
```

Comment:
Looks like this is generating a SARIF file for each story. When consuming SARIF files, do people normally get one file per thing that's tested, or one file per run of the tool?

Wondering if we want this to generate a single file for the test run, or if one per story is what we'd expect.

Reply:
Yeah, this PR definitely inspired #8.

That PR will probably take a bit of time to get reviewed and be released (probably days to a week or so). Is it reasonable to wait that long to recreate your SARIF functionality?

If you need/want that functionality ASAP, we can work on getting this in.